python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -93,6 +93,21 @@ def checkDiskQuota():
 
     return
 
+def bootstrap_reg_names():
+    # ALEX added this as need to ensure that prep registry is started up BEFORE job or template
+    # or even named templated registries as the _auto__init from job will require the prep registry to
+    # already be ready. This showed up when adding the named templates.
+    def prep_filter(x, y):
+        if x.name == 'prep':
+            return -1
+        return 1
+
+    names = []
+    for registry in sorted(getRegistries(), prep_filter):
+        names.append(registry.name)
```

Comment:
This code seems to be duplicated from the `bootstrap` function. Can that other function be made to call this one?